templates/Includes/DateRange.ss

```diff
@@ -1,6 +1,6 @@
 <div class="page-sidebar-widget news-events-date-range">
     <% with $DateRangeForm %>
-        <h3 class="h4 page-sidebar-header"><%t CWP\\CWP\\PageTypes\\EventHolder.FilterDate "Filter by date" %></h3>
+        <h3 class="h4 page-sidebar-header"><%t CWP\\CWP\\PageTypes\\EventHolder.FilterDate "Date" %></h3>
```

Comment:
Usually we don't change translation strings, because they will have already been translated from "Filter by date" to other languages in Transifex. I'd suggest if you want this to just say "Date" then also rename the key to "...EventHolder.Date" as well